packages/graphql-transformers-e2e-tests/src/__tests__/AuthV2ExhaustiveT2A.test.ts

```diff
@@ -10,8 +10,8 @@ import {
 
 const strategyProviders: Record<AuthStrategy, AuthProvider[]> = {
   owner: [],
-  public: ['apiKey', 'iam'],
```

Comment:
Aren't tests in the "graphql-transformers-e2e-tests" package V1 tests? Should we be asserting `identityPool` support for V1 transformer?

Reply:
> Aren't tests in the "graphql-transformers-e2e-tests" package V1 tests?

They're testing both. See https://github.com/aws-amplify/amplify-category-api/blob/main/packages/graphql-transformers-e2e-tests/package.json#L36-L58 . The tests I modified say `AuthV2` - presumably v2 auth transformers.

> Should we be asserting identityPool support for V1 transformer?

No. This is out of scope (see PR description and comments here https://app.asana.com/0/0/1206486113291629/1206699612098204/f )